tests/queries/0_stateless/03015_optimize_final_rmt.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: long, no-random-settings, no-random-merge-tree-settings
+# Tags: long, no-random-settings, no-random-merge-tree-settings, no-shared-merge-tree
```

Comment:
Why doesn't this test pass for SMT? I'd expect them to have matching behaviour

Reply:
it is not yet implemented for set